### PR TITLE
jquery for page on load and viewFavorites

### PR DESCRIPTION
**Pull request by kimalajoy**
### Type of change made:
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Styling- no new features
- [ ] Test
- [x] Refactoring

### Detailed Description
Refactored the on page load and view favorite functions into jQuery. Added the domUpdates.js file and moved the dom manipulation parts of 2 functions to that file.

### Why is this change required? What problem does it solve?
No particular change being required, besides the refactoring itself

### Were there any challenges that arose while implementing this feature? If so, how were the addressed?
Not 100% sure what that domUpdate file is for, I guess

### Files modified:
- [ ] index.html
- [ ] styles.css
- [ ] cookbook.js
- [ ] pantry.js
- [ ] recipe.js
- [ ] user.js
- [x] scripts.js
- [ ] pantry-test.js
- [ ] recipe-test.js
- [ ] user-test.js
- [x] Other files: domUpdates.js

### CODE REVIEW

File: src/scripts.js [hunk 1]
```diff
@@ -131,33 +116,18 @@ function getFavorites() {
...
-  $('.all-cards').html('');
-  if ($('.all-cards[class="all"]')) {
-    $('.all-cards').removeClass('all')
+  cardArea.empty();
+  if (cardArea.hasClass('all')) {
+    cardArea.removeClass('all');
   }
+
   recipes.forEach(recipe => {
-    $('.all-cards').append(`<div id='${recipe.id}'
```
Comment (kimalajoy) [inline src/scripts.js]: This .append part of the function was moved to the domUpdates.js file, but functionality still remains

File: src/scripts.js [hunk 2]
```diff
@@ -131,33 +116,18 @@ function getFavorites() {
   if (user.favoriteRecipes.length) {
     user.favoriteRecipes.forEach(recipe => {
       $(`.favorite${recipe.id}`).addClass('favorite-active')
-    })
-  } else return
```
Comment (kimalajoy) [inline src/scripts.js]: I removed this extra else statement as it's not needed

File: src/scripts.js [hunk 3]
```diff
@@ -4,25 +4,23 @@ import './css/styles.scss';
...
 import ingredientData from './data/ingredients';
 import users from './data/users';
-
 import Pantry from './pantry';
 import Recipe from './recipe';
 import User from './user';
 import Cookbook from './cookbook';
+import domUpdates from './domUpdates';
 
-let favButton = document.querySelector('.view-favorites');
```
Comment (kimalajoy) [inline src/scripts.js]: Updated the query selectors to jQuery

File: src/scripts.js [hunk 4]
```diff
@@ -4,25 +4,23 @@ import './css/styles.scss';
 import recipeData from './data/recipes';
 import ingredientData from './data/ingredients';
 import users from './data/users';
-
 import Pantry from './pantry';
 import Recipe from './recipe';
 import User from './user';
 import Cookbook from './cookbook';
+import domUpdates from './domUpdates';
```
Comment (tanishalatoya) [inline src/scripts.js]: Thanks for catching that we needed a separate domUpdates file. Would love to hear more about how you decided what should go into that file vs. the regular scripts file in our work session tomorrow.

Comment (tanishalatoya) [inline src/scripts.js]: Thanks for catching that we needed a separate domUpdates file. Would love to hear more about how you decided what should go into that file vs. the regular scripts file in our work session tomorrow.

Comment (tanishalatoya) [inline src/scripts.js]: Thanks for catching that we needed a separate domUpdates file. Would love to hear more about how you decided what should go into that file vs. the regular scripts file in our work session tomorrow.

File: src/scripts.js (hunk 1 above)
Comment (tanishalatoya) [reply src/scripts.js]: Looks good!

**Pull request merged**